esss_fix_format/cli.py

```diff
@@ -218,6 +218,10 @@ def get_files(cmd):
     result.update(get_files('git diff --name-only --diff-filter=ACM --staged'))
     result.update(get_files('git diff --name-only --diff-filter=ACM'))
     result.update(get_files('git ls-files -o --full-name --exclude-standard'))
+    # check_output returns bytes in Python 3
+    if sys.version_info[0] == 3:
```

Comment:
Not a big issue, but is is slightly better to check for `sys.version_info[0] > 2`, since it is more future-proof